test/jdk/tools/jlink/basic/AllModulePath.java

```diff
@@ -153,6 +252,21 @@ private void checkModules(Path image, Set<String> modules) throws Throwable {
                     .shouldHaveExitValue(0);
     }
 
+    /*
+     * Verify linked modules using java --list-modules
+     */
+    private void verifyListModules(Path targetPath, List<String> expected) throws Exception {
```

Comment:
```suggestion
    private void verifyListModules(Path targetPath, Set<String> expected) throws Exception {
```

Can simply use a Set to compare the result and  no need to sort `actual` from the output.

Reply:
OK.